forge-gui/res/cardsfolder/upcoming/abstruse_appropriation.txt

```diff
@@ -0,0 +1,10 @@
+Name:Abstruse Appropriation
+ManaCost:2 W B
+Types:Instant 
+K:Devoid
+A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Permanent.nonLand | RememberChanged$ True | TgtPromt$ Select target nonland permanent | SubAbility$ DBEffect | SpellDescription$ Exile target nonland permanent. 
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ MayPlay,ManaConvert | SubAbility$ DBCleanup | Duration$ Permanent | ForgetOnMoved$ Exile | SpellDescription$ You may cast that card for as long as it remains exiled, and you may spend colorless mana as though it were mana of any color to cast that spell.
+SVar:MayPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may cast that card for as long as it remains exiled
+SVar:ManaConvert:Mode$ ManaConvert | ValidPlayer$ You | ValidCard$ Card.IsRemembered | ValidSA$ Spell | ManaConversion$ AnyType->AnyColor | AffectedZone$ Exile | Description$ You may spend colorless mana as though it were mana of any color to cast that spell.
```

Comment:
you need to bind the convert to the casting permission
try with `Spell.MayPlaySource`

Reply:
@tool4ever @tehdiplomat maybe we should refactor this part?
https://github.com/Card-Forge/forge/blob/c0d1046326d694ef2126c18490a5434058008c13/forge-game/src/main/java/forge/game/card/CardPlayOption.java#L69-L78

Reply:
maybe but I'm not touching that until some other things with mayPlay have progressed further :p